egs2/TEMPLATE/enh1/enh.sh

```diff
@@ -71,6 +71,7 @@ is_tse_task=false   # Whether perform the target speaker extraction task or norm
 use_dereverb_ref=false
 use_noise_ref=false
 extra_wav_list= # Extra list of scp files for wav formatting
+use_utt2category=false
```

Comment:
Maybe we can follow the same style used in enh_asr1, without explicitly having this flag variables? Because I think it is a bit redundant.

Reply:
Sure. I will remove it.